<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sgt. Duffy</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/Duffy.svg</normaloff>:/icons/Duffy.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="showing">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Showing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="knownToggle">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>known</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>unknown</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="fileType">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>files</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>executables</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="changedSince">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>changed since</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Filename</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Modified</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MD5 hash</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>613</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp_index"/>
    <addaction name="actionAbout_Duffy"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionClose"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/Document-new.svg</normaloff>:/icons/Document-new.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/Document-open.svg</normaloff>:/icons/Document-open.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/Document-save.svg</normaloff>:/icons/Document-save.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/Document-save-as.svg</normaloff>:/icons/Document-save-as.svg</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/Document-close.svg</normaloff>:/icons/Document-close.svg</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/System-log-out.svg</normaloff>:/icons/System-log-out.svg</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionHelp_index">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/Help-browser.svg</normaloff>:/icons/Help-browser.svg</iconset>
   </property>
   <property name="text">
    <string>Help index</string>
   </property>
  </action>
  <action name="actionAbout_Duffy">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/Duffy.svg</normaloff>:/icons/Duffy.svg</iconset>
   </property>
   <property name="text">
    <string>About Duffy</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
